components/docs-chef-io/content/automate/infra_server.md

```diff
@@ -126,6 +126,27 @@ Installations require elevated privileges, so run the commands as the superuser
 
 1. [Set up `knife`]({{< relref "infra_server.md#use-knife-with-chef-infra-server" >}}) for use with Chef Infra Server.
 
+1. To send data from the Chef Infra Server to an external Chef Automate installation,
```

Comment:
I deliberately wrote up data collection as a separate step (rather than as part of the deploy config) because it will make debugging issues with data collector config a bit easier.

Reply:
Thanks!